jsr352/core/src/main/resources/META-INF/batch-jobs/hibernate-search-mass-indexing.xml

```diff
@@ -9,6 +9,12 @@
     xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/jobXML_1_0.xsd" version="1.0">
 
     <listeners>
+        <listener ref="org.hibernate.search.jsr352.massindexing.JobParameterValidationListener">
+            <properties>
+                <property name="checkpointInterval" value="#{jobParameters['checkpointInterval']}?:200;" />
+                <property name="rowsPerPartition" value="#{jobParameters['rowsPerPartition']}?:250;" />
```

Comment:
It weird to have a `rowsPerPartition` value which is not the multiple of `checkpointInterval`. However, since it's not part of the mission of this ticket, maybe we can handle it later.